cmake_modules/shares.cmake

```diff
@@ -0,0 +1,51 @@
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems�.
```

Comment:
new files require current year in copyright header

Reply:
added